engine/src/main/java/org/terasology/editor/properties/SceneProperties.java

```diff
@@ -30,9 +31,9 @@
     @Override
     public List<Property<?>> getProperties() {
         List<Property<?>> result = Lists.newArrayList();
-        Skysphere skysphere = CoreRegistry.get(WorldRenderer.class).getSkysphere();
-        if (skysphere != null) {
-            result.addAll(new ReflectionProvider(skysphere).getProperties());
+        BackdropProvider backdropProvider = CoreRegistry.get(BackdropProvider.class);
+        if (backdropProvider != null) {
+            result.addAll(new ReflectionProvider(backdropProvider).getProperties());
```

Comment:
This is part of a mechanism for making values available for tweaking in runtime when running TeraEd.